docs/DeveloperGuide.md

```diff
@@ -247,49 +247,105 @@ tagged with an existing module.
 will cause more time wasted for the user to fix them anyway, it seems that it is better to incur some overhead to
 prevent making a mess altogether.
 
-### Countdown feature \[coming in v1.3] (Wen Ling)
-TR4CKER has a countdowns tab which allows users to add important events that they would like TR4CKER to countdown to.
+### Countdown feature (Wen Ling)
+TR4CKER has a Countdown tab which allows users to add important events that they would like TR4CKER to countdown to.
 This feature allows users to isolate the most important time sensitive events and deadlines, and tells the user exactly
 how many days do they have to a certain event, which enhances the tracking experience.
 
-The 3 main functions of the Countdown feature are to:
-1. Display a list of all countdown events
-2. Display prominently the 2 earliest upcoming events
+These are the main functions of the Countdown feature:
+1. Display a list of all events
+2. Display the number of days left to an event,as well as the date of an event
```

Comment:
maybe can be `Display the number of days left to an event and the date of an event`?